apps/python/descriptions/scalert.xml

```diff
@@ -85,6 +85,18 @@
 					received pick has one of the value(s).
 					</description>
 				</parameter>
+				<parameter name="phaseNumber" type="int" default="1">
+					<description>
+					Start the pick script only when a minimum number of phases
+					'phaseNumber' is received within 'phaseInterval'.
+					</description>
+				</parameter>
+				<parameter name="phaseInterval" type="int" default="1" unit='s'>
+					<description>
```

Comment:
The default must be 0 or you change the behaviour of the module previously running elsewhere.

Reply:
There is no risk of that happening. The new parameters are effective only if the configured `phaseNum` is greater than 1, so the old behaviour is preserved unless the user make use of the new options.